lale/operators.py

```diff
@@ -452,6 +453,63 @@ def pretty_print(
             markdown = IPython.display.Markdown(f"```python\n{result}\n```")
             return IPython.display.display(markdown)
 
+    def diff(
+        self,
+        other: Union[Any, "Operator"],
```

Comment:
Why is this a `Union[Any, "Operator"]`? I would have expected just `"Operator"`.